lib/SGN/Controller/Locus.pm

```diff
@@ -93,22 +93,22 @@ sub view_by_name : Path('/locus/view/') Args(0) {
     my $locus_id = undef;
     my $locus = undef;
     if ($symbol && $species) { 
-	$locus = $c->stash->{locus} = CXGN::Phenome::Locus->new_with_symbol_and_species($c->dbc->dbh, $symbol, $species);
+        $locus = $c->stash->{locus} = CXGN::Phenome::Locus->new_with_symbol_and_species($c->dbc->dbh, $symbol, $species);
     }
 
-    if ($locusname) { 
-	$locus = $c->stash->{locus} = CXGN::Phenome::Locus->new_with_locusname($c->dbc->dbh, $locusname);
+    if ($locusname) {
+        $locus = $c->stash->{locus} = CXGN::Phenome::Locus->new_with_locusname($c->dbc->dbh, $locusname);
     }
     
-    if (defined($locus->get_locus_id())) { 
-	my $locus_id = $locus->get_locus_id();
-	$c->stash->{locus} = $locus;
-	$c->detach("/locus/view/", [ $locus_id] );    #("/locus/$locus_id/view");    
```

Comment:
where was this not working?

Reply:
On our wheat instance: 
https://wheat.triticeaetoolbox.org/locus/view?symbol=1RS1AL&species=Wheat 

we get the error:
`[error] Couldn't detach to command "/locus/view/": Invalid action or component.`

Reply:
ok, I see. I don't think we use this piece of code. We use the locus name as the identifier in the URL 
https://solgenomics.net/locus/Solyc01g005000/view

https://github.com/solgenomics/sgn/blob/master/lib/SGN/Controller/Locus.pm#L261